umh.docs.umh.app/content/en/docs/production-guide/backup_recovery/backup-restore-database.md

```diff
@@ -57,98 +64,145 @@ machine:
+   {{% /notice %}}
 
-{{< notice warning >}}
-This section is untested. Please report any issues you encounter.
-{{< /notice >}}
+3. Run the following command to connect to the factoryinsight database:
 
-For this section, we assume that you are restoring the data to a fresh United
-Manufacturing Hub installation with an empty database.
+   ```bash
+   psql "postgres://factoryinsight:<password>@<server-IP>:5432/factoryinsight?sslmode=require"
+   ```
 
-### Copy the backup file to the database pod
+   The default password is `changeme`.
 
-1. Open {{< resource type="lens" name="name" >}}.
-2. Launch a new terminal sesstion by clicking on the **+** button in the
-   bottom-left corner of the window.
-3. Run the following command to copy the backup file to the database pod:
+4. Check the table list running `\dt` and run the following command for each table 
+to save all data to `.csv` files:
 
-   ```bash
-   kubectl cp /path/to/local/backup.bak {{< resource type="pod" name="database" >}}:/tmp/backup.bak
+   ```sql
+   \COPY (SELECT * FROM <TABLE_NAME>) TO <TABLE_NAME>.csv CSV
    ```
 
-   Replace `/path/to/local/backup.bak` with the path to the backup file on your
-   machine.
+### Grafana and umh_v2 database
+
+If you want to backup the Grafana or umh_v2 database, you can follow the same steps 
+as above, but you need to replace any occurence of `factoryinsight` with `grafana`.
+
+In addition, you need to write down the credentials in the
+{{< resource type="secret" name="grafana" >}} Secret, as they are necessary
+to access the dashboard after restoring the database.
+
+The default username for `umh_v2` database is `kafkatopostgresqlv2`, and the password is 
+`changemetoo`. 
+
+## Restoring the database
+
 
-This step could take a while depending on the size of the backup file.
+For this section, we assume that you are restoring the data to a fresh United
+Manufacturing Hub installation with an empty database.
 
 ### Temporarly disable kafkatopostrgesql
 
+Connect to your server via SSH and run the following command: 
+
 <!-- tested in e2e #1343 -->
 ```bash
 sudo $(which kubectl) scale deployment {{< resource type="deployment" name="kafkatopostgresql" >}} --replicas=0 -n united-manufacturing-hub --kubeconfig /etc/rancher/k3s/k3s.yaml
 ```
 
-### Open a shell in the database pod
+### Restore the database
 
-{{< include "open-database-shell" >}}
+This section shows an example for restoring factoryinsight. If you want to restore 
+`grafana`, you need to replace any occurence of `factoryinsight` with `grafana`.
 
-### Restore the database
+For `umh_v2`, you should use `kafkatopostgresqlv2` for the user name and 
+`changemetoo` for the password.
+
+1. Make sure that your device is connected to server via SSH and run the following command:
+
+   {{< include "open-database-shell" >}}
 
-1. Drop the existing database:
+2. Drop the existing database:
 
    ```sql
-   DROP DATABASE factoryinsight;
+   DROP DATABASE factoryinsight WITH (FORCE);
```

Comment:
We might want to not do this, but instead include instructions on how to turn off kafkatopostgresql & kafkatopostgresqlv2 while dropping, as they might be writing actual data !

Reply:
kafkatopostgresql, kafkatopostgresqlv2, and factoryinsight are disabled while dropping now.